file.Rmd

```diff
@@ -0,0 +1,92 @@
+# Exploring the Recorded Crime Datasets
+This page will explore the three data sets of recorded crime in London, broken down by borough, ward and lower layer super output area (LSOA). These crimes are recorded using the [Home Office crime classifications](https://www.met.police.uk/sd/stats-and-data/met/crime-type-definitions/).
+
+```{python}
+import matplotlib.pyplot as plt
+import pandas as pd
+from projtools.data import Data
+
+data = Data()
+
+```
+
+#### Borough Dataset
+
+```{python}
+data.borough.head()
+```
+
+#### Ward Dataset
+
+```{python}
+data.ward.head()
+```
+
+#### LSOA Dataset
+
+```{python}
+data.lsoa.head()
+```
+
+<img src="../assets/london-borough-map.pjpeg" width=550 height=550>
+
+These are the 32 boroughs in London. Note that the City of London is not present as this borough has its own police force.
+
+```{python}
+bor_names = {'boroughs_borough': list(data.borough.index.get_level_values(2).unique().sort_values()),
+             'boroughs_ward': list(data.ward.index.get_level_values(4).unique().sort_values())}
+bors = pd.DataFrame(data=bor_names)
+bors
+```
+
+All the datasets contain a borough column, which for the `ward` and `lsoa` datasets refer to the borough in which the ward or LSOA sit.
+
+The dataframe above shows the values in the borough columns for the `borough` and `ward` datasets. As you can see here, there are *33* boroughs in each of the data frames, instead of _32_.
+
+In the `borough` data set there is an additional 'London Heathrow and London City Airports' value, while in the `ward` data set there is an 'Aviation Security (SO18)' value. The implications of this are discussed further on in the notebook.
+
+
+## Categories and Offences
+There are 12 categories, and 56 offences across those categories.
+
+```{python}
+list(data.borough.index.get_level_values(0).unique().sort_values())
+```
```

Comment:
Need help with doing this due to multi-index please @TomWFox

Reply:
![CleanShot 2023-12-19 at 22 14 24@2x](https://github.com/lisds/project-daylight-robbery/assets/13188249/a80f5047-9384-4017-ab7f-3f4218af5636)

I've done it like this, it's not perfect but I think it does the job nicely. Okay with you?